initial/src/main/java/com/example/springboot/HelloController.java

```diff
@@ -1,14 +1,40 @@
 package com.example.springboot;
 
+import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.bind.annotation.RequestMapping;
+import java.util.Random;
 
 @RestController
 public class HelloController {
 
 	@RequestMapping("/")
-	public String index() {
-		return "Hello Ethan Wei, welcome to Spring Boot!";
+	//create algorithm under controller to send the calculated value to the server
+	public String index(String[] args) {
+		Random rand = new Random();
+		int upperbound = 25;
+		int num1 = rand.nextInt(upperbound);
+		int num2 = rand.nextInt(upperbound);
+		String[] randOpr = {"+","-","*","/"};
+		int opr = rand.nextInt(randOpr.length);
+		String operand = randOpr[opr];
+		String equation = num1 + operand + num2;
+		int ans = applyOp(operand, num1, num2);
+		return equation + " = " + ans;
+	}
+	public static int applyOp(String operand, int num1, int num2) {
```

Comment:
1. There should always be a space between two different methods. 
2. Can you add [java doc](https://www.jrebel.com/blog/tips-and-tricks-for-better-java-documentation) for this method as you make this public? If this method is public, it can potentially be referenced at other places (either the same project or other projects using your project as a jar), so you should tell the caller what does this method do and how to use it.